src/drawing/DrawingAnnotations.tsx

```diff
@@ -47,6 +48,7 @@ const DrawingAnnotations = (props: Props): JSX.Element => {
         setStatus,
         setupDrawing,
         stashedPathGroups,
+        stroke: creatorStroke,
```

Comment:
Is the rename necessary? Does it conflict with `Stroke`? If so, maybe we can alias the type and keep this as stroke?

Reply:
No, it conflicts with the stroke in render `{drawnPathGroups.map(({ clientId: pathGroupClientId, paths, stroke }) => (...`